demo-app/src/app/form-validation/form-validation.component.html

```diff
@@ -0,0 +1,192 @@
+    as validator function in form control contructor. At the moment there is no method in Angular 2 to get the validator function list applied on a from control.
+  </p>
+</div>
+
+<div class="section">
+  <h5 class="light">Playground</h5>
+
+  <p>
+    This is a demo that demonstrate the form validation with <code class="language-markup">mz-validation</code> and Reactive Form. Here a list of each field with validation :
+  </p>
+
+  <ul class="browser-default">
+    <li>Name: Required, min length and max length</li>
+    <li>Last Name: Required</li>
+    <li>Address 1: Required</li>
+    <li>Province: Required</li>
+    <li>Postal Code : Pattern</li>
+    <li>Hear about us: Required</li>
+    <li>Term of service: Required</li>
+  </ul>
+
+  <div class="card">
+    <div class="card-tabs col s12">
+      <ul class="tabs">
+        <li class="tab"><a class="active" href="#view">View</a></li>
+        <li class="tab"><a href="#html">HTML</a></li>
+        <li class="tab"><a href="#code">TS</a></li>
+      </ul>
+    </div>
+    <div class="card-content">
+      <div id="view" class="col s12">
+        <form [formGroup]="userForm" (ngSubmit)="onSubmit()" novalidate>
+          <div class="row">
+            <mz-input-container class="col s12 m6">
+              <input mz-input mz-validation required
+                id="input-firstname"
+                formControlName="firstName"
+                type="text"
+                [errorMessage]="formErrors.firstName"
+                [label]="'First Name'" />
+            </mz-input-container>
+            <mz-input-container class="col s12 m6">
+              <input mz-input mz-validation required
+                id="input-lastname"
+                formControlName="lastName"
+                type="text"
+                [errorMessage]="formErrors.lastName"
+                [label]="'Last Name'" />
+            </mz-input-container>
+          </div>
+          <div class="row">
+            <mz-input-container class="col s12 m12">
+              <input mz-input mz-validation required
+                id="input-address"
+                formControlName="address"
+                type="text"
+                [errorMessage]="formErrors.address"
+                [label]="'Address 1'" />
+            </mz-input-container>
+          </div>
+          <div class="row">
+            <mz-textarea-container class="col s12 m12">
+              <textarea mz-textarea
+                id="textarea-address2"
+                formControlName="address2"
+                [label]="'Address 2'"
+                [length]="255">
+              </textarea>
+            </mz-textarea-container>
+          </div>
+          <div class="row">
+            <mz-input-container class="col s12 m4">
+              <input mz-input
```

Comment:
Yes, you can use Validators.maxLength and it will show the error message when the field caracther is greate than the validation.